From 097389897279e9b5a16d773648b9f0ebdb1e9d6a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Pablo=20Su=C3=A1rez=20Hern=C3=A1ndez?=
 <psuarezhernandez@suse.com>
Date: Mon, 22 Feb 2021 11:39:19 +0000
Subject: [PATCH] Allow extra_filerefs as sanitized kwargs for SSH
 client

(cherry picked from commit 89f843398849633af52cceab2155e9cedf8ad3dd)
---
 salt/client/ssh/client.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/salt/client/ssh/client.py b/salt/client/ssh/client.py
index ae514d67b3..1bd1246118 100644
--- a/salt/client/ssh/client.py
+++ b/salt/client/ssh/client.py
@@ -60,6 +60,7 @@ class SSHClient(object):
             ('rosters', list),
             ('ignore_host_keys', bool),
             ('raw_shell', bool),
+            ('extra_filerefs', str),
         ]
         sane_kwargs = {}
         for name, kind in roster_vals:
-- 
2.30.1


